gui/slick/interfaces/default/testRename.tmpl

```diff
@@ -43,13 +63,13 @@
     <thead>
         <tr class="seasonheader" id="season-$cur_ep_obj.season">
             <td colspan="4">
-                <br/>
+            	<br />
```

Comment:
Yup. That was already there.
On Jan 22, 2015 11:27 AM, "Fernando" <notifications@github.com> wrote:

> In gui/slick/interfaces/default/testRename.tmpl
> <https://github.com/SiCKRAGETV/SickRage/pull/1219#discussion_r23366496>:
>
> > @@ -43,13 +63,13 @@
> >      <thead>
> >          <tr class="seasonheader" id="season-$cur_ep_obj.season">
> >              <td colspan="4">
> > -                <br/>
> > +            	<br />
>
> is this right? space between br and / ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SiCKRAGETV/SickRage/pull/1219/files#r23366496>.
>